### PR TITLE
Add nested Activities on Plans

### PR DESCRIPTION
**Pull request by alexrudall**
- Add Cocoon Gem
- Add nested activities
- Add helpers to convert selected day to IceCube schedule

Screenshot 2020-10-23 at 18 52 15

Not done yet:
- Style activities form properly
- Start/end time

### CODE REVIEW

File: app/views/plans/index.html.erb [hunk 1]
```diff
@@ -34,10 +34,10 @@
                     <%= plan.name %>
                   </td>
                   <td>
-                    <%= plan.start_date %>
+                    <%= plan.start_date.strftime(I18n.t('date.formats.readable')) %>
```
Comment (barbarawuk) [inline app/views/plans/index.html.erb:37]:
```suggestion
                    <%= I18n.l(plan.start_date, format: :readable) %>
```

File: app/views/plans/index.html.erb [hunk 2]
```diff
@@ -34,10 +34,10 @@
...
                   </td>
                   <td>
-                    <%= plan.end_date %>
+                    <%= plan.end_date.strftime(I18n.t('date.formats.readable')) %>
```
Comment (barbarawuk) [inline app/views/plans/index.html.erb:40]:
```suggestion
                    <%= I18n.l(plan.end_date, format: :readable) %>
```

File: app/views/plans/index.html.erb (hunk 1 above)
Comment (barbarawuk) [reply app/views/plans/index.html.erb:37]: a little neater?

**Review by barbarawuk — COMMENTED** [review]
A couple of minor suggestions.
Haven't looked too closely at the schedule code though, can we go through it in a call?

File: app/views/plans/index.html.erb (hunk 1 above)
Comment (alexrudall) [reply app/views/plans/index.html.erb:37]: A lot neater! That's very cool.

**Pull request merged**